### PR TITLE
Add Experian credentials

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

**Review by subenson — DISMISSED** [review]

File: openapi/paths/credential-hashes@experian@{hash}.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            $ref: ../components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+patch:
+  tags:
+    - Email Credentials
```
Comment (VladimirBerdnik) [inline openapi/paths/credential-hashes@experian@{hash}.yaml:33]:
```suggestion
    - External Services integrations
```

File: openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    description: Experian credential
+    required:
+      - username
+      - password
+      - hmacKey
+      - publicKey
+    properties:
+      username:
+        type: string
+        description: Username on the Experian side
```
Comment (ashkarpetin) [inline openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml:13]: Punctuation: https://docs.microsoft.com/en-us/style-guide/punctuation/periods

e.g. https://github.com/Rebilly/api-definitions/pull/254/files

Comment (VladimirBerdnik) [reply openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml]: Fixed, thanks!

**Review by ashkarpetin — DISMISSED** [review]

**Review by slavcodev — DISMISSED** [review]

File: openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml (hunk 2 above)
Comment (vykhrystiuk) [reply openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml]: I think we should fix all descriptions:
https://github.com/Rebilly/api-definitions/pull/258/files#diff-f9df4409a43f752b0d3315b9f999c060016474b69d198320f79e57daf67c2d8fR9
https://github.com/Rebilly/api-definitions/pull/258/files#diff-f9df4409a43f752b0d3315b9f999c060016474b69d198320f79e57daf67c2d8fR77
...

Comment (VladimirBerdnik) [reply openapi/components/schemas/Credentials/ExperianCredentials/ExperianCredential.yaml]: @vykhrystiuk , I feel like dots in these places may be unnecessary 
image

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/paths/credential-hashes@experian@{hash}.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+get:
+  tags:
+    - Experian credentials
+  summary: Retrieve an Experian credential
+  operationId: GetExperianCredentialHash
+  description: |
+    Retrieve an Experian credential with specified token identifier string
+  responses:
+    '200':
+      description: AWS SES credential was retrieved successfully
```
Comment (adamaltman) [inline openapi/paths/credential-hashes@experian@{hash}.yaml:13]:
```suggestion
      description: Experian credential was retrieved successfully
```

**Review by ashkarpetin — APPROVED** [review]

**Comment by VladimirBerdnik** [comment]
devs please review

**Comment by VladimirBerdnik** [comment]
devs please review

**Review by 2e3s — APPROVED** [review]

**Pull request merged**